spinnaker_camera_driver/cmake/DownloadSpinnaker.cmake

```diff
@@ -3,13 +3,19 @@ function(download_spinnaker FLIR_LIB_VAR FLIR_INCLUDE_DIR_VAR)
     message(FATAL_ERROR "Downloading libSpinnaker for non-linux systems not supported")
   endif()
 
+  execute_process(
+    COMMAND lsb_release -cs
+    OUTPUT_VARIABLE OS_CODE_NAME
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
   include(cmake/TargetArch.cmake)
   target_architecture(FLIR_ARCH)
-  message(STATUS "Running download_spinnaker script with arguments: ${FLIR_ARCH} ${CATKIN_DEVEL_PREFIX}/lib/spinnaker_camera_driver/ WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}")
+  set(FLIR_DIR ${CMAKE_CURRENT_BINARY_DIR}/usr/lib)
```

Comment:
why wouldn't we use the library directory?

Reply:
Because this is the folder that the original version of the script used.
These modifications were made from the perspective of "the endpoint where we download the packages from is different; fix that, but don't modify the end behaviour of anything else unless it's strictly necessary."